renderer/src/hooks/StationRewards.tsx

```diff
@@ -1,5 +1,6 @@
 import { useEffect, useMemo, useState } from 'react'
 import { getScheduledRewards } from 'src/lib/station-config'
+import { sumAllRewards } from 'src/lib/utils'
```

Comment:
could you please move this function to this file? Since it's only used in one place, I would rather prevent `utils` from becoming a big utility bag, and have it here